scripts/globals/spells/invisible.lua

```diff
@@ -15,19 +16,14 @@ end
 function onSpellCast(caster, target, spell)
     if not target:hasStatusEffect(dsp.effect.INVISIBLE) then
 
-        -- last 7-9 minutes
-        local duration = calculateDuration(math.random(420, 540), spell:getSkillType(), spell:getSpellGroup(), caster, target)
-        duration = calculateDurationForLvl(duration, 25, target:getMainLvl())
+        local duration = calculateDuration(math.random(42, 54) * 10, spell:getSkillType(), spell:getSpellGroup(), caster, target)
```

Comment:
This is making the duration be locked to 10 second intervals. If that's intentional, I'm not sure where the proof for that behavior exists.